### PR TITLE
TSS-2191: Account home template

### PR DESCRIPTION
**Pull request by EPedley**

### CODE REVIEW

File: users/profile/forms/policy_teams.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+from django import forms
+
+
+class UserEditPolicyTeamsForm(forms.Form):
+    form = forms.CharField(
+        required=False,
+    )
+    label = "Label"
+    help_text = "Help text"
```
Comment (Santino-Trade) [inline users/profile/forms/policy_teams.py:11]: Think the label and help text need to be more appropriate - dont these get picked up by screen readers?

File: templates/users/policy_teams/edit_policy_teams.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+{% block body_script %}
+    {{ block.super }}
+    <script nonce="{{request.csp_nonce}}">
+        ma.components.MultiSelect();
+    </script>
+{% endblock %}
+
+{% block page_title %}{{ block.super }} - Account{% endblock %}
+{% block masthead %}
+    {% include 'users/partials/account/banner.html' with active="my_profile" %}
```
Comment (Santino-Trade) [inline templates/users/policy_teams/edit_policy_teams.html:14]: Remove the underscore from the active parameter for display heading

**Review by osimuka — APPROVED** [review]

**Review by ferozerub — APPROVED** [review]

File: users/profile/forms/policy_teams.py (hunk 1 above)
Comment (EPedley) [reply users/profile/forms/policy_teams.py]: All done

File: core/frontend/src/js/components/MultiSelect.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
+ma.components.MultiSelect = function (select_element) {
+    const add_button = document.getElementById("add_button");
```
[the remark this answers is not in the document]
Comment (EPedley) [reply core/frontend/src/js/components/MultiSelect.js:2]: WIP

**Pull request merged**